blocks/hero/index.js

```diff
@@ -0,0 +1,188 @@
+								<div className='travel-hero-search-dates flex my2 justify-around'>
+									<label className='travel-date-input relative bold flex-auto'  >
+										<input className='block relative p0 z1' type='date' placeholder='yyyy-mm-dd' pattern='[0-9]{4}-[0-9]{2}-[0-9]{2}' title='yyyy-mm-dd' name='departure' />
+										<svg className='travel-icon' viewBox='0 0 100 100'><path fill='currentColor' d='M7.93 79.476h84.32v8.876H7.93v-8.876zm86.848-41.538c-.932-3.55-4.615-5.68-8.165-4.704l-23.566 6.302L32.427 11l-8.566 2.263 18.374 31.82-22.056 5.902-8.743-6.834L5 45.883l8.077 14.023 3.417 5.903 7.1-1.91 23.566-6.3 19.305-5.148 23.565-6.302c3.594-1.02 5.68-4.66 4.748-8.21z'></path></svg>            <div className='travel-date-input-label'>
+										Departure
+									</div>
+									</label>
+									<label className='travel-date-input relative bold flex-auto' >
+										<input className='block relative p0 z1' type='date' placeholder='yyyy-mm-dd' pattern='[0-9]{4}-[0-9]{2}-[0-9]{2}' title='yyyy-mm-dd' name='return' />
+										<svg className='travel-icon' viewBox='0 0 100 100'><path fill='currentColor' d='M7.929 79.476h84.32v8.876H7.929v-8.876zm81.693-15.409c1.03-3.523-1.03-7.246-4.576-8.238L61.6 49.094 50.051 8.863l-8.508-2.471-.64 36.737-21.946-6.3-3.974-10.361-6.407-1.831-.3 16.18-.11 6.82 7.069 2.021 23.445 6.735 19.199 5.53 23.445 6.736c3.607.976 7.269-1.069 8.298-4.592z'></path></svg>            <div className='travel-date-input-label'>
+										Return
+									</div>
+									</label>
+								</div>
+
+								<a href='travel-results.amp' className='ampstart-btn travel-input-big rounded center bold white block col-12'>
+									Find Adventures &amp; Tours
+								</a>
+								<a className='travel-hero-discover block center mx-auto mt1 md-hide lg-hide'>
+									Explore <svg className='travel-icon' viewBox='0 0 66 100'><path fill='currentColor' d='M33.5 56.172l-18.96-18.1c-1.497-1.43-3.922-1.43-5.418 0a3.539 3.539 0 0 0 0 5.17l21.67 20.687a3.914 3.914 0 0 0 2.708 1.07c.98 0 1.96-.357 2.71-1.07l21.668-20.687a3.541 3.541 0 0 0 0-5.172c-1.496-1.427-3.92-1.427-5.417 0L33.5 56.173z'></path></svg>
+								</a>
+							</div>
+						</div>
+					</div>
+				</section>
+			);
+		},
+		save: props => {
+
+			// Note: amp-mustache template content is not included in "save".
+			const ampValueProp = {
+				'[value]': 'fields_query'
+			};
+
+			// @todo Confirm possible solution for replacing [src] -- add data-ampsrc instead and replace it with filters when displaying the post.
+			const ampSrcProp = {
+				'data-ampsrc': "'/api/places?types=(regions)&amp;types=(cities)&amp;input=' + fields_query_live"
+			};
```

Comment:
React is allowing the rest of the amp attributes to be saved (`[value]` etc.) but `[src]` not. Added a workaround which uses `data-ampsrc` instead and then this will later get replaced with `the_content` filter when rendering the post. Does this seem reasonable?

Reply:
This is essentially what we're having to do in PHP as well to support these attributes. Before sending into `DOMDocument` to parse, we replace all such attributes with `data-` attributes and then after parsing is done we convert them back to the original bracketed-notation. See: https://github.com/Automattic/amp-wp/blob/2bcfc19c2108b9b7bdd5c2eab1532e90a2bb0882/includes/utils/class-amp-dom-utils.php#L182-L272

I think what you have is more or less what we have to live with until AMP adds support for an XHTML syntax: https://github.com/ampproject/amphtml/issues/11115